rocky/reports/views/mixins.py

```diff
@@ -268,6 +276,9 @@ def save_report(self, report_names: list) -> Report:
         # Save the child reports to bytes
         for ooi, types in report_data.items():
             for report_type, data in types.items():
-                save_report_raw(bytes_client, data=data | self.get_input_data())
+                bytes_client.upload_raw(
```

Comment:
This needs some changes in base.py, get_input_data has "input_data"
![image](https://github.com/user-attachments/assets/a05ade78-5060-4e79-9518-a4c1b1c5fc29)

Reply:
I will fix this